doc-requirements.txt

```diff
@@ -4,6 +4,3 @@
 sphinx==3.5.4
 sphinx-autobuild
 sphinx-click
-scikit-learn
-scipy
-kubernetes
```

Comment:
Renamed this file to `doc-requirements.txt` since it better describes what this files contains.